src/Models/SeatGroup.php

```diff
@@ -20,87 +20,93 @@
 class Seatgroup extends Model
 {
     use Corporation;
+
     protected $fillable = ['name','description','type','role_id' ];
 
-    public function role(){
+    public function role()
+    {
         return $this->belongsToMany('Seat\Web\Models\Acl\Role');
     }
 
-    public function group(){
+    public function group()
+    {
         return $this->belongsToMany('Seat\Web\Models\Group')
             ->withPivot('is_manager','on_waitlist');
     }
 
-    public function corporation(){
+    public function corporation()
+    {
         return $this->belongsToMany('Seat\Eveapi\Models\Corporation\CorporationInfo',
             'corporation_info_seatgroup','seatgroup_id', 'corporation_id');
     }
-    public function manager(){
 
+    public function manager()
+    {
         return $this->belongsToMany('Seat\Web\Models\Group')
             ->wherePivot('is_manager',1);
     }
-    public function member(){
 
+    public function member()
+    {
         return $this->belongsToMany('Seat\Web\Models\Group')
             ->wherePivot('on_waitlist',0);
     }
-    public function waitlist(){
 
+    public function waitlist()
+    {
         return $this->belongsToMany('Seat\Web\Models\Group')
             ->wherePivot('on_waitlist',1);
     }
 
-    public function isManager(){
-        if(in_array(Auth::user()->group->id , $this->manager->map(function($group) { return $group->id; })->toArray()) || Auth::user()->hasRole('Superuser')) {
-            return true;
-        }
-        return false;
+    public function isManager()
+    {
+        return in_array(auth()->user()->group->id , $this->manager->map(function($group) {
+                return $group->id;
+            })->toArray()) || auth()->user()->hasSuperUser();
     }
 
-    public function isAllowedToSeeSeatGroup(){
-
-        if(in_array(Auth::user()->group->main_character->corporation_id , $this->corporation->pluck('corporation_id')->toArray()) || Auth::user()->hasRole('seatgroups.edit')) {
+    public function isAllowedToSeeSeatGroup()
+    {
+        if (auth()->user()->hasSuperUser())
```

Comment:
i believe this if statement needs some curly parenthesis to return true?

Reply:
don't get the thing here. What do you mean ?

Reply:
ups should be: https://github.com/herpaderpaldent/seat-groups/pull/6/files/1b4a6bd1e92e45fad31a228a9fbb9263f4c06915#diff-2a009b09197595663d5e60c26dbff927R71

The thing is it reads right now:
```
public function isAllowedToSeeSeatGroup()
{
   if (auth()->user()->hasSuperUser())
      return true;

   {some other logic}
```
i would expect now that it return true everytime?

Reply:
it will return true only if current user is super.

Reply:
i am no software engineer so i always thought one needs a closure after the if statement, which will be executed if the statement is true. Seems this is different in php. Thank you for your feedback!

Reply:
code blocks are not mandatory. Short representation will always work, as longer as the thing to be done is on a single line.

That's working for `while`, `for`, `if`